Manifest.cs

```diff
@@ -34,7 +34,7 @@
 [assembly: Feature(
     Id = FormConsent,
     Name = "Lombiq Privacy - Form consent",
-    Description = "Provides Consent Checkbox widget that can attach to any form.",
+    Description = "Provides the Consent Checkbox widget that be used on any form.",
```

Comment:
Már (és ismét bennem sem lehet bízni :D):
```suggestion
    Description = "Provides the Privacy Consent Checkbox widget that can be used on any form.",
```